editoast/src/views/project.rs

```diff
@@ -76,8 +76,8 @@ pub enum ProjectError {
 }
 
 /// Creation form for a project
-#[derive(Serialize, Deserialize, Derivative, ToSchema)]
-#[derivative(Default)]
+#[derive(Serialize, Deserialize, Educe, ToSchema)]
+#[educe(Default)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize, Default, ToSchema)]
```

There are other places where we can apply this change.